public/app/plugins/panel/canvas/editor/layerEditor.tsx

```diff
@@ -68,14 +67,6 @@ export function getLayerEditor(opts: InstanceState): NestedPanelOptions<LayerEdi
         // TODO: the non-root nav option
       }
 
-      builder.addCustomEditor({
-        id: 'content',
-        path: 'root',
-        name: 'Elements',
-        editor: LayerElementListEditor,
-        settings: { scene, layer: scene.currentLayer, selected },
-      });
-
```

Comment:
I think we will want to put the tree view nav here (to basically just replace the layerelementlisteditor in place) (as we will still want the background / border options for the root element found on the `layerEditor`
- With that change we could remove the separate `Tree View` editor section in both the inline editor / normal editor

I think we can also remove the related code to the `LayerElementListEditor` as well as part of this PR